data/projects/implement_aspects.md

```diff
@@ -1,30 +1,69 @@
-In the future, coala will choose the bears to run for the user based on what
-aspects of a programming language he wants to analyze. This project should
-realize this.
+Initial configuration of coala is **unnecessary hard**, especially for new
+user. There are few reason for this.
 
-The aspects concept exists as
-[cEP-0005](https://github.com/coala/cEPs/blob/master/cEP-0005.md) and it is
+New user is not always familiar with all the functionality provided by various
+tools inside coala. Besides that, coala tries to provide a full set of options
+for those tools, which could make configuring each tools overwhelming.
+There is the possibility, that the same type of checking is executed by
```

Comment:
Finally, there is the possibility..., which makes choosing the right tools for a type of analysis hard.